pkg/registry/core/adapters/ns_contex_client_test.go

```diff
@@ -0,0 +1,95 @@
+// Copyright (c) 2020 Doc.ai and/or its affiliates.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package adapters_test
+
+import (
+	"context"
+
+	"google.golang.org/grpc"
+
+	"github.com/networkservicemesh/sdk/pkg/registry/core/adapters"
+	"github.com/networkservicemesh/sdk/pkg/registry/core/next"
+	"github.com/networkservicemesh/sdk/pkg/registry/core/streamcontext"
+
+	"testing"
+
+	"github.com/golang/protobuf/ptypes/empty"
+	"github.com/networkservicemesh/api/pkg/api/registry"
+)
+
+type contextKeyType string
+
+const testKey contextKeyType = "PassingContext"
+
+// NewNSClient - returns NetworkServiceRegistryClient that checks the context passed in from the previous NSClient in the chain
+//             t - *testing.T used for the check
+//             check - function that checks the context.Context
+func NewNSClient(t *testing.T, check func(*testing.T, context.Context)) registry.NetworkServiceRegistryClient {
```

Comment:
I think we need to move this thing to `reigstry/utils/checks/checkcontext` to make it consistent with similar thing from `networkservice/`  https://github.com/networkservicemesh/sdk/blob/master/pkg/networkservice/utils/checks/checkcontext/client.go

Also, I'm ok if you make this thing private.

Reply:
ok, fixed